assignments/week_1/project1.4/ReadMe.md

```diff
@@ -0,0 +1,123 @@
+# Project 1.4
+Welcome to this first robotics themed projects of the RoboJackets software training
+curriculum. The point of these projects is to give you hands on experience playing
+around with code related to a robotics topic that we have covered in lecture.
+I hope you enjoy this one
+
+# Project Objective
+During this project we will be training a Neural Network on the MNIST dataset.
+This is the canonical hello world of the machine learning world. The goal is to
+classify images of handwritten digits into what it is. We will be using a
```

Comment:
`The goal is to classify images of handwritten digits into the digits they represent`?